cmd/etcd.go

```diff
@@ -113,76 +115,79 @@ func showDDLInfoCommandFunc(cmd *cobra.Command, args []string) {
 	cmd.Println(res)
 }
 
-func delOwnerCampaign(cmd *cobra.Command, args []string) {
+func delKeyCommandFunc(cmd *cobra.Command, args []string) {
 	if len(args) != 1 {
-		cmd.Println(cmd.UsageString())
+		cmd.Printf("Only one argument!")
+		return
+	}
+	apiScope := "/tidb/ddl"
```

Comment:
Allow to delete key range in `/tidb/ddl` is dangarous.

Reply:
Do we need to delete key in `/tidb/ddl/fg/owner` ?